src/Config/EnvironmentData.php

```diff
@@ -136,31 +136,56 @@ public function getApplication(): array
             return $this->data['application'];
         }
 
-        $application = $this->getEnvVar(SystemConfigInterface::VAR_ENV_APPLICATION, []);
+        $applicationEnvConfig = $this->getEnvVar(SystemConfigInterface::VAR_ENV_APPLICATION, []);
+        $applicationFileConfig = $this->readApplicationConfig();
 
-        if (!$application) {
+        if (!$applicationEnvConfig) {
+            return $this->data['application'] = $applicationFileConfig;
+        }
+
+        /**
+         * Temporary fix for the case when environment data does not accurately represent file configuration.
+         *
+         * @url https://github.com/magento/magento-cloud-docker/issues/292
```

Comment:
Add to that issue to revert this tmp fix

Reply:
Done